docs/proposals/external-ca.md

```diff
@@ -0,0 +1,112 @@
+# External Certificate Authority Proposal
```

Comment:
Let's document that additionally we need to add support for Postgres over TLS (to secure CA cert and key in transit)